### PR TITLE
Add add127 documentation

### PR DESCRIPTION
**Pull request by XapaJIaMnu**
Is this good enough for #34  ?

TBH I'm not happy with the names **SomethingNew**, but I am not sure with what i should replace the **New** bit?

### CODE REVIEW

**Comment by XapaJIaMnu** [comment]
Also, adding a new postprocessing function is no longer very obvious, we should have a description for that.

**Comment by kpu** [comment]
Why not make it an entirely separate class like Int8Shift?

**Comment by XapaJIaMnu** [comment]
Like this?

File: intgemm.cc [hunk 1]
```diff
@@ -22,10 +22,12 @@ void (*Int8::PrepareB)(const float *input, int8_t *output, float quant_mult, Ind
 
 void (*Int8::SelectColumnsB)(const int8_t *input, int8_t *output, Index rows, const Index *cols_begin, const Index *cols_end) = ChooseCPU(AVX512_8bit::SelectColumnsB, AVX2_8bit::SelectColumnsB, SSSE3_8bit::SelectColumnsB, Unsupported_8bit::SelectColumnsB, Unsupported_8bit::SelectColumnsB);
 
-//void (*Int8::PrepareBiasFor8)(const float *input, float *bias, float alpha, Index rows, Index cols) = ChooseCPU(AVX512_8bit::PrepareBiasFor8, AVX2_8bit::PrepareBiasFor8, SSSE3_8bit::PrepareBiasFor8, Unsupported_8bit::PrepareBiasFor8, Unsupported_8bit::PrepareBiasFor8);
-
 const char *const Int8::kName = ChooseCPU(AVX512_8bit::kName, AVX2_8bit::kName, SSSE3_8bit::kName, Unsupported_8bit::kName, Unsupported_8bit::kName);
 
+void (*Int8Shift::QuantizeU)(const float *input, uint8_t *output, float quant_mult, Index size) = ChooseCPU(AVX512_8bit::QuantizeU, AVX2_8bit::QuantizeU, SSSE3_8bit::QuantizeU, Unsupported_8bit::QuantizeU, Unsupported_8bit::QuantizeU);
```
Comment (kpu) [inline intgemm.cc]: We can probably afford more of a name than QuantizeU

**Pull request merged**